samples/src/JustSaying.Sample.Restaurant.OrderingApi/JustSaying.Sample.Restaurant.OrderingApi.csproj

```diff
@@ -9,6 +9,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Serilog.AspNetCore" Version="3.2.1-dev-00142" />
+    <PackageReference Include="Serilog.Sinks.Seq" Version="4.0.0" />
```

Comment:
Did you mean to include this package, or was this for local debugging?

Reply:
Yes, have been using seq to analyse the behaviour of the new consumer bits. Previously the internal library code used SerilogTimings, but I'm going to pull that out to an `IMessageMonitor` now that we support that.

Seq dashboards via structured logging are an excellent way to get metrics on consumption rates/handler durations/receive timings without depending on something like statsd.

Reply:
I'd argue this should be left in, because the behaviour of a non-existent server at this address is just to backoff expontentially, it becomes a no-op, and the value is auto-sending logs in a local dev environment.

 I'm sure you're aware of all this 🙂